app/src/main/java/skills/future/planer/SplashActivity.java

```diff
@@ -0,0 +1,28 @@
+package skills.future.planer;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import android.view.View;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class SplashActivity extends AppCompatActivity {
```

Comment:
a ta klasa to do czego?

Reply:
Splash screen - ekran ładowania aplikacji